### PR TITLE
[IMP] top_bar: css tweaking

### PR DESCRIPTION
**Pull request by pro-odoo**
Task-id 2883223

## Description:

description of this task, what is implemented and why it is implemented that way.

Odoo task ID : [2883223](https://www.odoo.com/web#id=2883223&action=333&active_id=2328&model=project.task&view_type=form&cids=1&menu_id=4720)

## review checklist

- [ ] feature is organized in plugin, or UI components
- [ ] support of duplicate sheet (deep copy)
- [ ] in model/core: ranges are Range object, and can be adapted (adaptRanges)
- [ ] in model/UI: ranges are strings (to show the user)
- [ ] undo-able commands (uses this.history.update)
- [ ] multiuser-able commands (has inverse commands and transformations where needed)
- [ ] new/updated/removed commands are documented
- [ ] exportable in excel
- [ ] translations (\_lt("qmsdf %s", abc))
- [ ] unit tested
- [ ] clean commented code
- [ ] track breaking changes
- [ ] doc is rebuild (npm run doc)
- [ ] status is correct in Odoo

### CODE REVIEW

File: src/components/top_bar/top_bar.ts [hunk 1]
```diff
@@ -86,6 +86,12 @@ interface Props {
   onComposerContentFocused: (selection: ComposerSelection) => void;
 }
 
+const FONT_COLOR = "#4a4f59";
+const ACTIVE_BG_COLOR = "#E6F4EA"; //"#f6f7fa";
+const ACTIVE_FG_COLOR = "#188038"; //"#000000";
```
Comment (pro-odoo) [inline src/components/top_bar/top_bar.ts:91]: Remove commented color

**Review by pro-odoo — COMMENTED** [review]
You can squash the two commits

**Comment by pro-odoo** [comment]
robodoo r+

**Comment by rrahir** [comment]
@robodoo r+

**Pull request closed without merging**